environment.yml

```diff
@@ -5,21 +5,25 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - aipy=3.0.0rc2
+  - aipy>=3.0
   - attrs
   - h5py
   - hdf5
   - healpy
-  - mpi4py
   - nose
   - numpy
   - pip
-  - pytest
-  - pyuvdata
+  - pytest-cov
   - scipy
   - pyyaml
+  - psutil
+  - astropy>=2.0.16 #IERS bug
+  - numba
   - future #only for python 2
   - pip:
     - cached-property
-    - pyuvsim
+    - git+https://github.com/RadioAstronomySoftwareGroup/pyuvsim.git
+    - git+https://github.com/RadioAstronomySoftwareGroup/pyuvdata.git
     - git+https://github.com/HERA-Team/uvtools.git
+    - git+https://github.com/radioastronomysoftwaregroup/healvis 
+    - astropy-healpix
```

Comment:
Is this required? Shouldn't it be automatically installed via `pyuvdata`?

Reply:
I think we found in #63 that we need to include `astropy-healpix` in the environment, otherwise it never actually gets installed.

Reply:
ugh. OK.